core/shared/src/main/scala/com/lynbrookrobotics/potassium/units/Histogram.java

```diff
@@ -0,0 +1,54 @@
+package com.lynbrookrobotics.potassium.units;
```

Comment:
That is not the correct package to put this in. I think `commons` may be a better place.